GAMES_LIST.md

```diff
@@ -460,6 +461,10 @@ Conan Exiles never responds to player query.
 ### <a name="minecraft">Minecraft
 Many Minecraft servers do not respond with players data.
 
+### <a name='farmingsimulator'>Farming Simulator
+Farming Simulator 2022 servers need a token (reffered as code in the game). It can be obtained at your server's web interface (http://ip:port/settings.html). It can be passed to GameDig with the additional option: `token`
```

Comment:
`Farming Simulator 2022 servers need a token`, so, only FS22 or both? as this note is added on both of them.